File: pinyougou-web-shop/src/main/webapp/js/controller/seckillOrderController.js
//控制层
app.controller('seckillOrderController' ,function($scope,$controller,$location,seckillOrderService){

    $controller('baseController',{$scope:$scope});//继承

    $scope.search = function (page,rows) {
        seckillOrderService.search(page,rows).success(
            function (response) {
                $scope.list=response.rows;
                $scope.paginationConf.totalItems=response.total;//更新总记录数
            }
        )
    }
})

File: pinyougou-web-shop/src/main/java/cn/itcast/core/controller/seckill/SeckillOrderController.java
package cn.itcast.core.controller.seckill;

import cn.itcast.core.entity.PageResult;
import cn.itcast.core.service.seckill.SeckillOrderService;
import com.alibaba.dubbo.config.annotation.Reference;
import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * @Description:
 * @Author: zc
 * @CreateDate: 2019/2/14$ 10:49$
 */
@RestController
@RequestMapping("/order")
public class SeckillOrderController {

    @Reference
    private SeckillOrderService seckillOrderService;

    @RequestMapping("/search.do")
    public PageResult search(Long page,Long rows){
        String sellerId = SecurityContextHolder.getContext().getAuthentication().getName();

        return seckillOrderService.search(page,rows,sellerId);
    }
}
